radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/ForksModule.java

```diff
@@ -86,13 +83,13 @@ protected void configure() {
 
   @Provides
   @Singleton
-  private Forks forks(TreeMap<Long, ForkConfig> forkConfigs) {
+  private Forks forks(NavigableMap<Long, ForkConfig> forkConfigs) {
```

Comment:
why is it necessary?

Reply:
TreeMap is a concrete class, NavigableMap is an interface. Basically I've just fixed one of the code smells reported by Sonar.

Reply:
ah cool, I didn't remember that.